.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: ruby
+rvm:
+  - 2.5.0
+
+services:
+  - postgresql
+
+before_install:
+  - gem update --system
+
+before_script:
+  - psql -c "CREATE USER \"cq-rails\" WITH PASSWORD 'cq-rails';" -U postgres
+  - psql -c "DROP DATABASE IF EXISTS \"cq-rails_test\";" -U postgres
+  - psql -c "CREATE DATABASE \"cq-rails_test\" ENCODING 'utf8';" -U postgres
+  - bundle exec rails db:migrate
+
+script:
+  # - bundle exec rspec spec -fd
```

Comment:
Are we sure we want to avoid running rspec with Travis???

Reply:
I just comment it out because I haven't installed rspec so far. I was planning to uncomment it as soon as I get to a card were I get to install rspec.

Reply:
Great!